test/core/end2end/end2end_tests.cc

```diff
@@ -406,8 +406,9 @@ std::vector<CoreEnd2endTestRegistry::Test> CoreEnd2endTestRegistry::AllTests() {
   }
   for (const auto& suite_configs : suites_) {
     if (suite_configs.second.empty()) {
-      CrashWithStdio(
-          absl::StrCat("Suite ", suite_configs.first, " has no tests"));
+      fprintf(
```

Comment:
Please update to gpr_log

Reply:
needs to work without gpr_log initialized (that's why we used `CrashWithStdio` previously too)